lib/Backend/GlobOpt.cpp

```diff
@@ -7184,13 +7198,36 @@ GlobOpt::SetSymStoreDirect(ValueInfo * valueInfo, Sym * sym)
     if (prevSymStore && prevSymStore->IsStackSym() &&
         prevSymStore->AsStackSym()->HasByteCodeRegSlot())
     {
-        Assert(this->blockData.changedSyms != nullptr);
-        this->blockData.changedSyms->Set(prevSymStore->m_id);
+        this->SetChangedSym(prevSymStore->m_id);
     }
     valueInfo->SetSymStore(sym);
 }
 
 void
+GlobOpt::SetChangedSym(SymID symId)
+{
+    GlobOptBlockData * globOptData = nullptr;
+    if (this->currentBlock->globOptData.changedSyms)
+    {
+        globOptData = &this->currentBlock->globOptData;
+    }
+    else
```

Comment:
This looks correct.

---
In reply to: [72362653](https://github.com/Microsoft/ChakraCore/pull/1337#discussion_r72362653) [](ancestors = 72362653,72358009,72355295)